src/main/drivers/timer_hal.c

```diff
@@ -764,6 +778,9 @@ static void timCCxHandler(TIM_TypeDef *tim, timerConfig_t *timerConfig)
 
 #if USED_TIMERS & TIM_N(1)
 _TIM_IRQ_HANDLER(TIM1_CC_IRQHandler, 1);
+#if defined(STM32H7)
```

Comment:
Technically this should go into TIM10 unused branch or  USED_TIMERS & TIM_N(10) should be `#elif`
Also note used indentation of `# if`

Reply:
Hmm... H7 doesn't have TIM1_UP_TIM10 interrupt, so it should be ...
```
#if USED_TIMERS & TIM_N(1)
_TIM_IRQ_HANDLER(TIM1_CC_IRQHandler, 1);
#  if defined(STM32H7)
_TIM_IRQ_HANDLER(TIM1_UP_IRQHandler, 1);
#  else
#    if USED_TIMERS & TIM_N(10)
_TIM_IRQ_HANDLER2(TIM1_UP_TIM10_IRQHandler, 1, 10);  // both timers are in use
#    else
_TIM_IRQ_HANDLER(TIM1_UP_TIM10_IRQHandler, 1);     // timer10 is not used
#    endif
#  endif
#endif
```
Thoughts?

Reply:
Yes, this is IMO better ...